### DISCUSSION TITLE
Can I join team?

### DISCUSSION THREAD

**Discussion by ghost** [ROM-builders/temporary#13517] [General]
Device Tree: https://github.com/a3project/android_device_xiaomi_rosy
Kernel Tree: https://github.com/a3project/android_kernel_xiaomi_rosy
Vendor Tree: https://github.com/a3project/android_vendor_xiaomi_rosy

Tg: https://t.me/aepranata

Latest successfull build; https://cirrus-ci.com/build/5718949894553600

**Comment by ghost** [comment]
@Apon77

**Reply by Apon77** [reply]
Username or name in telegram and github does not match

**Reply by ghost** [reply]
Yes, but same person. I can confirm that

**Comment by ghost** [comment]
![Screenshot_20220924-135757](https://user-images.githubusercontent.com/108798043/192084653-47e2417a-a8e4-4a1c-a218-2a0656f87b04.png)

**Comment by Apon77** [comment]
i cant find you when i need to mention you in telegram!

**Comment by Apon77** [comment]
most importantly, i dont believe some one name should not exist project in his name!
